lib/src/sql/value/value.rs

```diff
@@ -150,6 +151,7 @@ pub enum Value {
 	Constant(Constant),
 	// Closure(Box<Closure>),
 	Function(Box<Function>),
+	MlModel(Box<MlModel>),
```

Comment:
New variants should go to the bottom of the enum, unless if this requirement changed with the introduction of versioned serialisation.